cmd/deploy.go

```diff
@@ -184,6 +184,10 @@ func deployFn(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
+	if err := c.PrepareDNSServers(); err != nil {
```

Comment:
let's rename to `ExtractDNSServers`

As for the calling this in deploy I think it can be moved to NewContainerlab? After the config is initialized.

Reply:
Done